src/ctap/data_formats.rs

```diff
@@ -367,16 +368,21 @@ impl TryFrom<cbor::Value> for GetAssertionHmacSecretInput {
                 1 => key_agreement,
                 2 => salt_enc,
                 3 => salt_auth,
+                4 => pin_uv_auth_protocol,
             } = extract_map(cbor_value)?;
         }
 
         let key_agreement = CoseKey::try_from(ok_or_missing(key_agreement)?)?;
         let salt_enc = extract_byte_string(ok_or_missing(salt_enc)?)?;
         let salt_auth = extract_byte_string(ok_or_missing(salt_auth)?)?;
+        let pin_uv_auth_protocol = pin_uv_auth_protocol
+            .map(PinUvAuthProtocol::try_from)
+            .unwrap_or(Ok(PinUvAuthProtocol::V1))?;
```

Comment:
Those 2 lines could be `.map_or(Ok(V1), try_from)?`.

Reply:
Changed here and in another unrelated occasion.